docs/_posts/examples/3400-01-18-playback-locations.html

```diff
@@ -0,0 +1,179 @@
+---
+layout: example
+category: example
+title: Play map locations as a slideshow
+description: 'Autoplay the locations of boroughs in New York City'
+tags:
+  - animation
+---
+<style>
+.map-overlay-container {
+    position: absolute;
+    width: 25%;
+    top: 0;
+    left: 0;
+    padding: 10px;
+    z-index: 1;
+}
+
+.map-overlay {
+    font: 12px/20px 'Helvetica Neue', Arial, Helvetica, sans-serif;
+    background-color: #fff;
+    border-radius: 3px;
+    padding: 10px;
+    box-shadow:0 1px 2px rgba(0,0,0,0.20);
+}
+
+.map-overlay h2,
+.map-overlay p {
+    margin: 0 0 10px;
+}
+</style>
+
+<div id='map'></div>
+
+<div class='map-overlay-container'>
+  <div class='map-overlay'>
+    <h2 id='location-title'></h2>
+    <p id='location-description'></p>
+    <small>Text credit: <a href='http://www.nycgo.com/neighborhoods'>nycgo.com</a></small>
+  </div>
+</div>
+
+<script>
+mapboxgl.accessToken = 'pk.eyJ1IjoibWFwYm94IiwiYSI6ImdQMzI4WjgifQ.d-Uyr7NBjrJVz9z82uk5Xg';
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/emerald-v8',
+    center: [-74.0315, 40.6989],
+    maxZoom: 16,
+    minZoom: 9,
+    zoom: 9.68
+});
+
+var title = document.getElementById('location-title');
+var description = document.getElementById('location-description');
+
+var locations = [{
+    "title": "The Bronx",
+    "description": "This is where hip-hop was born, where the Yankees became a dynasty and where you can find New York City's leading zoo and botanical garden.",
+    "interaction": function(callback) {
```

Comment:
The structure of this callback is identical for every element, so make it data-driven and write the procedural code only once.

```
{
  "id": "2",
  "title": "The Bronx",
  "description": "...",
  "camera": {
    "center": [-73.8709, 40.8255],
    "zoom": 12.21,
    "pitch" 0
  }
}

function playback(index) {
  ...
  highlightBorough(locations[index].id);
  map.flyTo(locations[index].camera);
  ...
}
```